src/lints/pub_api_sealed_trait_became_unsealed.ron

```diff
@@ -0,0 +1,54 @@
+SemverQuery(
+    id: "pub_api_sealed_trait_became_unsealed",
+    human_readable_name: "Sealed trait in the public API became unsealed",
+    description: "An public API sealed trait is now unsealed, allowing downstream crates to implement it. Reverting this in the future would be a breaking change",
+    required_update: Minor,
+    lint_level: Warn,
+    reference_link: Some("https://rust-lang.github.io/api-guidelines/future-proofing.html#sealed-traits-protect-against-downstream-implementations-c-sealed"),
+    query: r#"
+    {
+        CrateDiff {
+            baseline {
+                item {
+                    ... on Trait {
+                        visibility_limit @filter(op: "=", value: ["$public"]) @output
+                        public_api_sealed @filter(op: "=", value: ["$true"])
+                        unconditionally_sealed @filter(op: "!=", value: ["$true"])
+
+                        importable_path {
+                            path @output @tag
+                            public_api @filter(op: "=", value: ["$true"])
+                        }
+                    }
+                }
+            }
+            current {
+                item {
+                    ... on Trait {
+                        visibility_limit @filter(op: "=", value: ["$public"])
+                        public_api_sealed @filter(op: "!=", value: ["$true"])
+                        name @output
+
+                        importable_path {
+                            path @filter(op: "=", value: ["%path"])
+                            public_api @filter(op: "=", value: ["$true"])
+                        }
+
+                        span_: span @optional {
+                            filename @output
+                            begin_line @output
+                            end_line @output
+                        }
+                    }
+                }
+            }
+        }
+    }"#,
+    arguments: {
+        "public": "public",
+        "true": true,
+    },
+    error_message: "An public API sealed trait is now unsealed, allowing downstream crates to implement it. Resealing this later would be a breaking change.",
```

Comment:
```suggestion
    error_message: "A public API sealed trait has become unsealed, allowing downstream crates to implement it. Reverting this would be a major breaking change.",
```